src/docker/processor/start.sh

```diff
@@ -13,4 +13,25 @@ if [[ "$HEALTHCHECK_BEFORE_START" == "true" ]];then
     done
 fi
 
-/usr/local/bin/processor -c /config/data/config.yaml
+psql $DATABASE_URL -c '\copy processor_status to /app/out.csv csv'
+
+if [ -s "out.csv" ];then
+    export STARTING_VERSION=$(cut -d, -f2 /app/out.csv)
```

Comment:
This should be `[ -s "/app/out.csv" ]`

Will fix in #657

Reply:
Fixed at https://github.com/econia-labs/econia/pull/657/commits/b3a36d4ae04e2fd18ad36e110da46d8b58903075